modules/ossm-migrating-complete-remove-2-6-control-plane.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/done/ossm-migrating-complete-assembly
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-remove-2-6-control-plane_{context}"]
+= Remove the Service Mesh 2.6 control plane
+
+Once you have migrated your deployment and workloads, you can remove {SMproduct} 2.x control plane.
+
+.Prerequisites
+
+* You have completed your deployment and workloads migrations.
+* You are logged in to the {ocp-product-title} web console as a user with the cluster-admin role.
+
+[NOTE]
+====
+Depending on how you created `ServiceMeshMembers` and `ServiceMeshMemberRoll` resources, those resources might be removed automatically with removal of the `ServiceMeshControlPlane` resource.
+====
+
+.Procedure
+
+. Find all Service Mesh 2.6 resources by running the following command:
++
+[source,terminal]
+----
+$ oc get smcp,smm,smmr -A
+----
+
+. Remove all found `ServiceMeshControlPlane` resources by running the following command:
++
+[source,terminal]
+----
+$ oc delete smcp basic -n istio-system
+----
+
+. Remove all found `ServiceMeshMemberRoll` resources by running the following command:
++
+[source,terminal]
+----
+$ oc delete smmr default -n istio-system
+----
+
+. Remove all found `ServiceMeshMembers` resources by running the following command:
++
+[source,terminal]
+----
+$ oc delete smm default -n bookinfo
```

Comment:
This is again just an example how to remove one

Reply:
oc delete smm --all -A

Reply:
we can replace this command